{{ cookiecutter.repo_name }}/Makefile

```diff
@@ -51,3 +52,13 @@ docs: ## Build docs using Sphinx and copy to docs folder (this makes it easy to
 
 ipython: ## Provides an interactive ipython prompt
 	docker exec -it $(CONTAINER_NAME) ipython
+
+setup-dvc: ## Initializes DVC in project repo
+	docker exec -it $(CONTAINER_NAME) dvc init
+	git add .dvc .dvcignore
```

Comment:
For one-off set-up commands (especially ones that run `git commits`), we probably want to move them out of the `Makefile` and into `hooks/post_gen_project.sh`.

E.g. this what we currently do at launch

<img width="510" alt="image" src="https://user-images.githubusercontent.com/12516153/103565935-0ca24280-4e76-11eb-916f-7000b3f53879.png">

I would suggest:
1. move lines 57-59 to `post_gen_project`. No more `setup-dvc`
2. Keep `setup-dvc`, but it only runs `docker exec -it $(CONTAINER_NAME) dvc init`. The post hook runs this command followed by the git commands.

Reply:
Agree this is definitely cleaner